### DISCUSSION TITLE
Specifying Additional OpenAPI Properties

### DISCUSSION THREAD

**Discussion by nullandvoid-digital** [vitalik/django-ninja#856] [Q&A] [answered]
I see the OpenAPI 3.0 spec has options like `x-logo` and `info`. Can these be passed to the API class to add something like a logo to the redoc page similar to passing the `title` property to the class?

**Comment by vitalik** [comment, marked as answer]
@PhoenixStorm1015 

you can pass openapi_extra into NinjaAPI instance:

https://github.com/vitalik/django-ninja/blob/7f5def3aeacf5500334604f028b22314a7e42eae/tests/test_openapi_extra.py#L28-L35

**Reply by nullandvoid-digital** [reply]
And the `openapi_extra` would take any of the valid properties listed in the specification, correct? Apologies if this is a silly question. The docs seem a little unclear on it and I'm new to this.
